src/callbacks_step/save_solution_dg.jl

```diff
@@ -73,6 +75,16 @@ function save_solution_file(u, time, dt, timestep,
             var = file["element_variables_$v"]
             attributes(var)["name"] = string(key)
         end
+
+        # Store node variables
+        for (v, (key, node_variable)) in enumerate(node_variables)
+            # Add to file
+            file["node_variables_$v"] = node_variable
+
+            # Add variable name as attribute
+            var = file["node_variables_$v"]
+            attributes(var)["name"] = string(key)
```

Comment:
```suggestion
            attributes(node_variable)["name"] = string(key)
```
Is this not the same?

Reply:
No, `node_variables` and `file["node_variables_$v"]` are not the same data type. Array vs. HDF5.Dataset